script/test

```diff
@@ -49,6 +49,22 @@ ECDSA_TEST_CERTIFICATE="${PROJECT_ROOT_DIR}/ecdsa.pem"
 
 CERTTOOL_BIN="certtool"
 
+build_test_plugins () {
```

Comment:
Yes, this is pretty much same as if server would fail to get certificate or some REST API response status would be wrong. It is that way to assure, that it is clear, how plugin implementation works.